### PR TITLE
Instance Announcements model.

### PR DESCRIPTION
**Pull request by lowjoel**
Like #49.

### CODE REVIEW

File: app/models/instance/announcement.rb [hunk 1]
```diff
@@ -0,0 +1,6 @@
+class Instance::Announcement < ActiveRecord::Base
+  stampable
+
+  belongs_to :creator, class_name: User.name
+  belongs_to :course, inverse_of: :announcements
```
Comment (wangqiang1208) [inline app/models/instance/announcement.rb]: You forgot to change here :-)

File: spec/factories/instance_announcements.rb [hunk 2]
```diff
@@ -0,0 +1,13 @@
+FactoryGirl.define do
+  factory :instance_announcement, class: Instance::Announcement.name do
+    course
```
Comment (wangqiang1208) [inline spec/factories/instance_announcements.rb]: and here

File: app/models/instance/announcement.rb (hunk 1 above)
Comment (lowjoel) [inline app/models/instance/announcement.rb]: thanks! haha I copy-pasted...

**Comment by lowjoel** [comment]
@jsyeo I think you can review this, thanks!

**Pull request merged**